### ISSUE TITLE
model.eval() needed when evaluating accuracy

### ISSUE THREAD

**Issue by shoryaconsul** [saksham36/Deep-Learning-Scaffold#1]
In _trainer.py_, model.eval() should be invoked before computing accuracy and model.train() called before training the model.
